.circleci/config.yml

```diff
@@ -0,0 +1,130 @@
+version: 2
+
+workflows:
+  version: 2
+  test-build-deploy:
+    jobs:
+    - lint
+    - test
+    - integration
+    - build
+    - deploy:
+        requires:
+        - build
+        - test
+        - lint
+        - integration
+
+jobs:
+  lint:
+    docker:
+    - image: tomwilkie/cortex-build-image:circleci-2.0-c80dbc66f
+
+    working_directory: /go/src/github.com/weaveworks/cortex
+    steps:
+    - checkout
+    - run:
+        name: Lint
+        command: |
+          touch build-image/.uptodate
+          make BUILD_IN_CONTAINER=false lint
+
+  test:
+    docker:
+    - image: tomwilkie/cortex-build-image:circleci-2.0-c80dbc66f
+
+    working_directory: /go/src/github.com/weaveworks/cortex
+    steps:
+    - checkout
+    - run:
+        name: Test
+        command: |
+          touch build-image/.uptodate
+          make BUILD_IN_CONTAINER=false test
+
+  integration:
+    docker:
+    - image: tomwilkie/cortex-build-image:circleci-2.0-c80dbc66f
+    - image: circleci/postgres:9.6.2-alpine
```

Comment:
What does it mean to have two `image`s in one section?  I can't figure it out from the docs.